git_guide.md

```diff
@@ -0,0 +1,22 @@
+# Git guide
+This guide is designed to help you get started with git and use its basics commands.
+
+- [Getting started](#getting-started)
+- [First contribution](#first-contribution)
+- [Good first bugs](#good-first-bugs)
+
+## Getting started
+To get started with git,first install git from the given link or if you have already installed it then update to its latest version.
+- [Install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+After installing git successfully, make a account at github.
+- [Get started with GitHub](https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account)
+
+## First contribution
+Making a contribution is always good way to start in GitHub.
+Let's make your first contribution using some simple steps and learning some commands.
+- [Make your contribution](https://github.com/firstcontributions/first-contributions)
```

Comment:
I appreciate that we're linking out to external resources rather than creating our own.